httpx/_transports/asgi.py

```diff
@@ -35,12 +42,19 @@ def create_event() -> "Event":
         return asyncio.Event()
```

Comment:
It looks to me like we can drop this `create_event()` function and the preceding `if typing.TYPE_CHECKING` block?